src/ir/from_ast/astconv.rs

```diff
@@ -834,6 +851,109 @@ impl<'ctx, 'prog> BuildCtx<'ctx, 'prog> {
     }
 }
 
+/// Build connect info between invokes and components
```

Comment:
This comment should be more detailed. What is "connect info" supposed to track and where is it used?